packages/state-transition/src/util/electra.ts

```diff
@@ -30,14 +31,20 @@ export function switchToCompoundingValidator(state: CachedBeaconStateElectra, in
 export function queueExcessActiveBalance(state: CachedBeaconStateElectra, index: ValidatorIndex): void {
   const balance = state.balances.get(index);
   if (balance > MIN_ACTIVATION_BALANCE) {
+    const validator = state.validators.getReadonly(index);
     const excessBalance = balance - MIN_ACTIVATION_BALANCE;
     state.balances.set(index, MIN_ACTIVATION_BALANCE);
 
-    const pendingBalanceDeposit = ssz.electra.PendingBalanceDeposit.toViewDU({
-      index,
-      amount: BigInt(excessBalance),
+    const pendingDeposit = ssz.electra.PendingDeposit.toViewDU({
+      pubkey: validator.pubkey,
+      withdrawalCredentials: validator.withdrawalCredentials,
+      amount: excessBalance,
+      // Use bls.G2_POINT_AT_INFINITY as a signature field placeholder
+      signature: new Uint8Array(G2_POINT_AT_INFINITY),
```

Comment:
can this `Uint8Array` be a const to reuse?

Reply:
> can this `Uint8Array` be a const to reuse?

Just realized Buffer extends from Uint8Array. Can use directly without instantiate a new Uint8Array